### DISCUSSION TITLE
Bechmarks with high communication

### DISCUSSION THREAD

**Discussion by Soilorian** [sarchlab/mgpusim#95] [General]
Hello Mr. Sun,
I hope you are well.

I am looking for some communication-intensive benchmarks that also support multiple GPUs. Could you suggest some existing benchmarks that meet these criteria? Additionally, could you guide me on the steps required to add a new benchmark to mgpusim?

**Comment by syifan** [comment]
My understanding is that most benchmarks are quite communication intensive if you use unified GPUs. 

To prepare new benchmarks, can you check this guide https://syifan.notion.site/MGPUSim-Preparing-New-Experiments-9b85ee8759064ddab8f0b745bab97057?pvs=4?
